engine.go

```diff
@@ -94,6 +94,18 @@ func (eng *Engine) Observe(name string, value interface{}, tags ...Tag) {
 	eng.measure(name, value, Histogram, tags...)
 }
 
+// Clock returns a new clock identified by name and tags.
+func (eng *Engine) Clock(name string, tags ...Tag) *Clock {
+	now := time.Now()
```

Comment:
We should copy the tags here, otherwise it may be non-intuitive that this slice gets captured by the returned Clock.

Reply:
good call, copied! :)